templates/ts-react-apollo-client/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.16.1",
   "private": true,
   "devDependencies": {
+    "@graphql-cli/codegen": "1.17.8",
```

Comment:
Actually we might try to not have cli here but all those packages in root. This way graphql-cli will be able to update them

Reply:
`graphql-codegen` works with graphqlrc files, so you could stick with that?